docs/modules/user-guide/partials/proc_limiting-resources-usage.adoc

```diff
@@ -86,7 +84,7 @@ This limit will be applied to every container of the given component.
 +
 For the `plugin` components, CPU limits can be described in the plug-in descriptor file, typically named `meta.yaml`.
 +
-If none of them are specified, system-wide defaults are applied (see description of `pass:[CHE_WORKSPACE_SIDECAR_DEFAULT__CPU__LIMIT__CORES]` system property).
+If none of them are specified, system-wide defaults are applied (see description of `pass:[WORKSPACE_SIDECAR_DEFAULT__CPU__LIMIT__CORES]` system property).
```

Comment:
I am not sure if this system property exists and how plugin behaves when no default is set in this case. @davidfestal @l0rd  We are trying to make the devfile spec Che independent. Do you have any suggestions on how to approach this?

Reply:
I think that this line should be removed. If the devfile doesn't specify components requests/limits, the client (`odo` or `che` for example) will decide if and how to infer a default value. And Kubernetes will use `LimitRanges` if those are specified in the target namespaces. From a devfile spec we should say that, if they are not specified, the values are undetermined: they may (or may not) be inferred from the application that consumes the devfile or from Kubernetes.